### ISSUE TITLE
[Common Lisp] Implement new Concept Exercise: characters

### ISSUE THREAD

**Issue by TheLostLambda** [exercism/v3#1776]
This issue describes how to implement the `characters` concept exercise for the Common Lisp track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Learning objectives

- Know how character literals can be written using the `#\c` syntax
- Be aware of the special `#\Space` and `#\Newline` characters
- Be able to apply simple predicates like `alphanumericp`, `digit-char-p`, `upper-case-p`, etc
- Know how to compare characters in a case-sensitive manner using `char>`, `char=`, etc
- Know how to compare characters in a **case-insensitive** manner using `char-lessp`, `char-equal`, etc
- Be able to convert between characters and their numerical representations using `char-code` and `code-char`

## Out of scope

- Strings
- Implementation-specific character sets
- Unicode

## Concepts

- `characters`

## Prerequisites

- `integers`
- `sameness`
- `truthy-and-falsy`

## Resources to refer to

- [Common Lisp the Language](https://www.cs.cmu.edu/Groups/AI/html/cltl/clm/node137.html)

### Hints

- [Practical Common Lisp](http://www.gigamonkeys.com/book/numbers-characters-and-strings.html)

### After

- [Unicode Support](https://www.cliki.net/Unicode%20support)
- [Unicode Library](http://edicl.github.io/cl-unicode/)

## Representer

Nothing yet

## Analyzer

Nothing yet

## Implementing

It's probably best to focus the exercise on what's standard, but it's certainly worth pointing out the widespread Unicode support of modern lisp implementations in the `after.md`

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
